### PR TITLE
Test `require` with string literals

### LINKED ISSUE #6651: [Sol -> Yul]: Add tests for `require()` when used with a string literal
Depends on #6650

### PR DESCRIPTION
**Pull request by Marenz**
fixes #6651

### CODE REVIEW

**Review by leonardoalt — DISMISSED** [review]

**Comment by Marenz** [comment]
> Test failing

Hmm not sure why. Locally it works

**Comment by Marenz** [comment]
@erak any idea why this fails in the CI but not locally?

File: test/libsolidity/semanticTests/viaYul/require.sol [hunk 1]
```diff
@@ -12,17 +12,25 @@ contract C {
 		x = true;
 		require(true);
 	}
-	/* Not properly supported by test system yet
 	function f2(bool a) public pure returns (bool x) {
 		x = a;
 		string memory message;
+		message = "fancy message!";
 		require(a, message);
```
Comment (axic) [inline test/libsolidity/semanticTests/viaYul/require.sol:19]: Can you keep a case with `string memory message; require(a, message);` to test empty strings?

**Comment by Marenz** [comment]
Addet test for empty string back

**Review by axic — DISMISSED** [review]
Looks good to me, but someone else should also check.

**Comment by leonardoalt** [comment]
Test failing

**Comment by Marenz** [comment]
retriggered tests

**Review by chriseth — APPROVED** [review]

**Pull request merged**